cypress/support/commands.js

```diff
@@ -183,16 +182,16 @@ Cypress.Commands.add('testDataTestidList', (dataTable, values) => {
 
 Cypress.Commands.add('addMockOrcid', (username) => {
   cy.window()
-  .its('store')
-  .invoke('getState')
-  .then((state) => {
-    const user_authority = state.user.authority;
-    user_authority.orcids.push('test_orcid');
-    cy.window().its('store').invoke('dispatch', {
-      type: 'set authority data',
-      authority: user_authority,
+    .its('store')
+    .invoke('getState')
+    .then((state) => {
+      const user_authority = state.user.authority;
```

Comment:
litt miks av navnekonvensjon her, siden det er `.js` foreslår jeg:
```suggestion
      const { authority } = state.user;
```
evt `const userAuthority = (...)`